src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static final int maxNumber = 100;
```

Comment:
```suggestion
    private static final int MAX_NUMBER = 100;
```
don't forget to use CONSTANT_CASE for naming of constants

Reply:
yeah... :)